packages/shared/components/organisms/ProposalCard.svelte

```diff
@@ -32,10 +32,12 @@
 
 <proposal-card
     on:click={handleProposalClick}
-    class="flex flex-col p-6 border border-solid border-gray-200 dark:border-transparent rounded-xl cursor-pointer h-32
-    {proposal.status === ProposalStatus.Ended ? 'bg-transparent' : 'bg-white dark:bg-gray-850'}"
+    on:keydown={(e) => e.key === 'Enter' && handleProposalClick()}
+    class="flex flex-col p-6 border border-solid border-gray-200 dark:border-transparent rounded-xl cursor-pointer h-fit shadow-elevation-1 hover:shadow-inner focus:shadow-inner
```

Comment:
I saw that this has been added to the task, but I'm not aware that we needed shadows. I asked design team to confirm this? I really dislike the hover shadow and we don't have it anywhere else as far as I'm aware

Reply:
I will remove the hover effect but I will keep the focus one, we need some visual guiding to the keyboard tabbing